ibis/backends/trino/__init__.py

```diff
@@ -112,3 +121,91 @@ def _execute_view_creation(self, name, definition):
 
         with self.begin() as con:
             con.execute(view)
+
+    def create_table(
+        self,
+        name: str,
+        obj: pd.DataFrame | pa.Table | ir.Table | None = None,
+        *,
+        schema: sch.Schema | None = None,
+        database: str | None = None,
+        temp: bool = False,
+        overwrite: bool = False,
+        comment: str | None = None,
+        properties: Mapping[str, Any] | None = None,
+    ) -> ir.Table:
+        if obj is None and schema is None:
+            raise com.IbisError("One of the `schema` or `obj` parameter is required")
+
+        if temp:
+            raise NotImplementedError(
+                "Temporary tables in the Trino backend are not yet supported"
+            )
+
+        orig_table_ref = self._fully_qualified_name(name, database)
+
+        if overwrite:
+            name = util.gen_name("trino_overwrite")
+
+        create_stmt = "CREATE"
+
+        table_ref = self._fully_qualified_name(name, database)
+
+        create_stmt += f" TABLE {table_ref}"
+
+        if schema is not None and obj is None:
+            schema_str = ", ".join(
+                (
+                    f"{name} {TrinoType.to_string(typ)}"
+                    + " NOT NULL" * (not typ.nullable)
+                )
+                for name, typ in schema.items()
+            )
+            create_stmt += f" ({schema_str})"
+
+        if comment is not None:
+            create_stmt += f" COMMENT {comment!r}"
+
+        if properties:
+            pairs = ", ".join(f"{k}={v!r}" for k, v in properties.items())
+            create_stmt += f" WITH ({pairs})"
```

Comment:
Do we need any quoting of properties here?

Same question for column/table names above.

Reply:
For `WITH` properties, I don't believe so. For column and table names, yes.

Reply:
Done. I overrode the `_fully_qualified_name` implementation and am quoting everything there.

Reply:
Since the default implementation of `_fully_qualified_name` doesn't do this, won't that potentially have repercussions in other methods? For example, the implementation of `BaseAlchemyBackend.drop_table` uses the fully qualified name in the `_schemas` mapping - moving to quoting that name might cause issues? I'd also worry that some backends handling the quoting in `_fully_qualified_name` (while others don't) might make it harder to know if a name is quoted or not.

Reply:
Potentially, but we don't have a test case that breaks after changing to this implementation.

Do you have anything in mind?

Reply:
`_fully_qualified_name` doesn't really have a contract around quoting as far as I can tell. Perhaps we should nail that down soon.

Reply:
I'm going to have to address this to enable cross schema table access, and I will definitely be changing the implementation of `_fully_qualified_name` to quote everything.

The current base implementation doesn't even do it what it says on the box, it just returns `name`.

Reply:
I'll propose that this particular thing seem like a thing that is too broken to quibble over.

Reply:
I'm not trying to "quibble over" this, I'm just pointing it out as a potential issue as part of the code review process. If the answer is that it's not an issue or that we don't worry about it now that's fine.

Reply:
Lemme see if in the meantime I can use the current broken implementation until I get around to address its issues.

Reply:
Ok, I just removed the use of `_fully_qualified_name` and noted that the `database` argument isn't yet supported.

Reply:
@jcrist Ah! I think my comment came across wrong! I only meant that I think I have some deeper work around that method to do and that it does need some kind of specification and a bit of thinking aloud that **I** am 100% willing to discard the changes to `_fully_qualified_name`!

Reply:
No worries!